.github/workflows/stubtest_third_party.yml

```
@@ -0,0 +1,70 @@
+name: Third-party stubtest
+
+on:
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - 'scripts/**'
+
+permissions:
+  contents: read
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  stubtest-third-party:
+    name: Check third party stubs with stubtest
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "windows-latest", "macos-latest"]
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Install dependencies
+        run: pip install -r requirements-tests.txt
+      - name: Run stubtest
+        shell: bash
+        run: |
+          STUBS=$(
+            git diff --name-only origin/${{ github.base_ref }} HEAD |
+            # Uncomment the following to (very slowly) run on all third party stubs:
+            # git ls-files |
```

This comment is somewhat out of date now; the `daily.yml` workflow is a much faster way of accomplishing the same thing

```suggestion
            # Use the daily.yml workflow to run stubtest on all third party stubs
```